ocaml/sdk-gen/go/gen_go_helper.ml

```diff
@@ -331,42 +376,65 @@ module Json = struct
        )
     |> String.concat ", "
 
+  let add_session_info method_name =
```

Comment:
How about moving these bits into the `addtion_infos_for_session`/`additional_info_of_session`. It would look more compact.